bootstrap/main.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import pathlib
+import shutil
+import sys
+import time
+from typing import Any
+
+import docker
+
+client = docker.from_env()
+
+default_file_path = pathlib.Path("/startup.json.default")
```

Comment:
global variables should be CAPITALIZED

Reply:
[show they](https://www.python.org/dev/peps/pep-0008/#global-variable-names)?

constants should, but I don't think I'd call these constants

Reply:
Wait, how `default_file_path` is not a constant ? Are you changing it somewhere ?

Reply:
we agreed that we will use CAPS as a "this should not be changed/mutated" indicator